tests-scan

```diff
@@ -241,6 +241,10 @@ def cockpit_tasks(api: github.GitHub, contexts: Sequence[str], opts: argparse.Na
                 env["BASE_BRANCH"] = branch
             if number:
                 env["TEST_PULL"] = f'{number}'
+            # for updating naughty trackers and downloading private images
+            secrets = ["github-token", "image-download"]
+            if api.repo == "rhinstaller/anaconda-webui":
```

Comment:
Maybe also try to limit this only to nightlies?

Reply:
It is a bit ad hoc, right. https://github.com/rhinstaller/anaconda-webui/pull/641 is still figuring out the details. I propose we start with that, and once @KKoukiou is happy with how it works, we further limit the secret by scenario or by branch name (only handing it out for testing the `main` branch -- can we do that? absent `number`?)